nodl_python/nodl/errors.py

```diff
@@ -122,26 +87,15 @@ class InvalidTopicError(InvalidElementError):
     """Base class for all errors in parsing a topic."""
 
 
-class AmbiguousTopicInterfaceError(InvalidTopicError):
-    """Error raised when a topic has no interface exposed."""
-
-    def __init__(self, element: etree._Element) -> None:
-        super().__init__(
-            f'Topic <{element.get("name")}> is neither publisher nor subscription', element=element
-        )
-
-
 class InvalidServiceError(InvalidElementError):
     """Base class for all errors in parsing a service."""
 
 
-class AmbiguousServiceInterfaceError(InvalidServiceError):
-    """Error raised when a topic has no interface exposed."""
+class InvalidNodeChildError(InvalidElementError):
+    """Error raised when a node has a child with an unsupported tag."""
 
     def __init__(self, element: etree._Element) -> None:
-        super().__init__(
-            f'Service <{element.get("name")}> is neither server nor client', element=element
-        )
+        super().__init__(f'Tag {element.tag} is not a valid child of a Node', element)
```

Comment:
Can you phrase this in a more... human-readable way? Can we write it in ROS terms instead of XML terms?

Reply:
Reworded in 42a43ab

Reply:
That looks great, thank you.